cekit-helpers/cekit-cache-server/src/main/java/org/kie/cekit/cacher/builds/nightly/NightlyBuildsPullRequestAcceptor.java

```diff
@@ -131,12 +110,12 @@ public void onFilePersisted(String fileName, String checkSum) {
 
                                 log.fine(String.format("Updating BC monitoring rhpam_business_central_monitoring_distribution.zip from [%s] to [%s]", artifact.getMd5(), bcMonitoringCheckSum));
                                 artifact.setMd5(bcMonitoringCheckSum);
-                                yamlFilesHelper.writeModule(bcMonitoring, bcMonitoringFile);
+                                yamlFilesHelper.writeModule( buildUtils.bcMonitoring(),  buildUtils.bcMonitoringFile());
 
                                 // find name: "rhpam_business_central_monitoring_distribution.zip"
                                 // and add comment on next line : rhpam-${version}.redhat-${buildDate}-monitoring-ee7.zip
                                 // or rhpam-${version}.PAM-redhat-${buildDate}-monitoring-ee7.zip depending on PAM version
-                                reAddComment(bcMonitoringFile, "name: \"rhpam_business_central_monitoring_distribution.zip\"",
+                                buildUtils.reAddComment( buildUtils.bcMonitoringFile(), "name: \"rhpam_business_central_monitoring_distribution.zip\"",
```

Comment:
formatting change: removing white space
```suggestion
                                buildUtils.reAddComment(buildUtils.bcMonitoringFile(), "name: \"rhpam_business_central_monitoring_distribution.zip\"",
```